frontend/app/routes/api/buildinfo.ts

```diff
@@ -6,8 +6,14 @@ import { TYPES } from '~/.server/constants';
  * An API endpoint that returns the build info.
  */
 export function loader({ context: { appContainer }, params, request }: LoaderFunctionArgs) {
-  const buildInfo = appContainer.get(TYPES.core.BuildInfoService).getBuildInfo();
-  const imageTag = `${buildInfo.buildVersion}-${buildInfo.buildRevision}-${buildInfo.buildId}`;
+  const serverConfig = appContainer.get(TYPES.configs.ServerConfig);
```

Comment:
Use `ClientConfig` to ensure no unwanted configs are returned to the client.

```suggestion
  const clientConfig = appContainer.get(TYPES.configs.ClientConfig);
```

Reply:
Since the configs are returns to the client then using `ClientConfig` make obvious what is the intention.